src/utils.tsx

```diff
@@ -309,3 +309,15 @@ export const parseOrderStatus = (status: OrderStatus) => {
       return '';
   }
 };
+
+/**
+ * Given a merch item, return the first picture associated with it.
+ */
+export const getDefaultMerchItemPicture = (item: PublicMerchItem | undefined): string | undefined => {
+  if (item && item.merchPhotos.length !== 0) {
+    // Get the item here with the smallest position (since it doesn't always arrive sorted)
+    item.merchPhotos.sort((a, b) => a.position - b.position);
+    return item.merchPhotos[0].uploadedPhoto;
```

Comment:
since you only need the image with the lowest position, sorting it is unnecessary

but whatever, this is frontend webdev, performance is an afterthought

Reply:
yeah its at max 5 times I think we will survive o7